core/pkg/eval/fractional_evaluation.go

```diff
@@ -22,7 +24,7 @@ func NewFractionalEvaluator(logger *logger.Logger) *FractionalEvaluator {
 	return &FractionalEvaluator{Logger: logger}
 }
 
-func (fe *FractionalEvaluator) FractionalEvaluation(values, data interface{}) interface{} {
+func (fe *FractionalEvaluator) FractionalEvaluation(values, data any) any {
```

Comment:
Since we are on Go 1.19 we can replace `interface{}` with `any`. I replaced all occurrences in this file.

Reply:
A definite readability improvement.